src/main/resources/logback-spring.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE configuration>
+
+<configuration scan="true">
+
+    <conversionRule conversionWord="clr" converterClass="org.springframework.boot.logging.logback.ColorConverter"/>
+    <conversionRule conversionWord="wex" converterClass="org.springframework.boot.logging.logback.WhitespaceThrowableProxyConverter"/>
+    <conversionRule conversionWord="wEx" converterClass="org.springframework.boot.logging.logback.ExtendedWhitespaceThrowableProxyConverter"/>
+
+    <property resource="META-INF/build-info.properties"/>
+    <property name="LOG_LOCATION" value="./logs"/>
+    <property name="C_LOG_PATTERN" value="${C_LOG_PATTERN:-%d{${LOG_DATEFORMAT_PATTERN:-yyyy-MM-dd HH:mm:ss.SSSXXX}} %clr(%5p) %clr([%X{traceId:-},%X{spanId:-}]){magenta} %clr(-&#45;&#45;){faint} %clr([%15.15t]){faint} %clr(%-40.40logger{39}){cyan} %clr(:){faint} %clr([%17X{stepType}]){blue} %X{requestTo:-} %X{requestContent:-} %X{responseContent:-} %X{responseCode:-} %X{responseInMs:-} %m%n${LOG_EXCEPTION_CONVERSION_WORD:-%wEx}}"/>
+    <property name="C_LOG_CHARSET" value="${C_LOG_CHARSET:-${file.encoding:-UTF-8}}"/>
+    <property name="F_LOG_PATTERN" value="${F_LOG_PATTERN:-%timeStamp ${build.version} %5p %X{requestAuthorIp:-} [%17X{stepType}] [%X{traceId:-},%X{spanId:-}] %X{requestTo:-} %X{requestContent:-} %X{responseContent:-} %X{responseCode:-} %X{responseInMs:-} : %m%n${LOG_EXCEPTION_CONVERSION_WORD:-%wEx}}"/>
```

Comment:
I like the way it's currently readable by using `[%17X{stepType}]` but we have to replace it with `{stepType}`. It makes logs a bit more messy (which can be overcome with whatever someone is using to read logs with), but `%17X` comes from the length of `incoming.response`, being currently the longest step name. But steps can be added in the future and this hard-coded value would become an issue.